docs/source/delta-uniform.md

```diff
@@ -103,15 +103,14 @@ In general, Iceberg and Delta table versions do not align by either the commit t
 
 ## Limitations
 
-.. warning:: UniForm is read-only from an Iceberg perspective, though this cannot yet be enforced since UniForm uses file-system-based commits when writing Iceberg. If any external writer (not <Delta>) writes to this Iceberg table, this may destroy your Delta table and cause data loss, as the Iceberg writer may perform data cleanup or garbage collection that Delta is unaware of.
+.. warning:: UniForm is read-only from an Iceberg perspective, though this cannot yet be enforced since UniForm uses HMS as catalog. If any external writer (not <Delta>) writes to this Iceberg table, this may destroy your Delta table and cause data loss, as the Iceberg writer may perform data cleanup or garbage collection that Delta is unaware of.
 
 The following limitations exist:
 
 - UniForm does not work on tables with deletion vectors enabled. See [_](delta-deletion-vectors.md).
-- Delta tables with UniForm enabled do not support `LIST`, `MAP`, and `VOID` types.
+- Delta tables with UniForm enabled do not support `VOID` type.
```

Comment:
Do we want to point to the allow list in delta protocol to make the description more accurate?

Reply:
I will skip for simplicity for most of users... expert users will check protocol anyway.